### ISSUE TITLE
Creation of a profile when using "zint" function of the plane_layer_volume tool

### ISSUE THREAD

**Issue by vbouillaut** [BenMql/coral#19]
Hi,

The function zint() of the plane_layer_volume package creates a profile + a verticalSums even when applied on a profile instead of only a verticalSums.

Vincent Bouillaut

**Comment by BenMql** [comment]
@vbouillaut ,
That is correct: applying the `zint` method of the `plane_layer_volume` class of `${CORAL}/etc/python_scripts/process_data.py` to a profile, say `p(z)`, computes two quantities:
+ An integrated profile `q(z) = int_{u=zmin}^{u=z} p(u) du` (with z being a point of the Gauss colocation grid)
+ the vertical sum `int_{u=zmin}^{u=zmax} p(u) du` 
I will leave this issue open until I have time to document this feature in the wiki.
